ci/cirrus-templates/script.bash

```diff
@@ -26,6 +26,7 @@ echo "Install Rustup using ./rustup-init.sh"
 
 sh rustup-init.sh --default-toolchain=stable --profile=minimal -y
 # It's the equivalent of `source`
+# shellcheck source=/dev/null
```

Comment:
ok so this one is failing because the script wants to import from a file that won't exist until rustup is installed. But this is linted *after* rustup is installed, so I would expect it to find the source.

Reply:
I think it only checks the source file from this repo. So we need to find the `env` file in the source code, but it is generated from `src/cli/self_update/env.sh`. And I tried set it in https://github.com/rust-lang/rustup/pull/3121/commits/d74db478af4085aff37bf9d1310d84db2b7c2b75.

It failed: https://github.com/rust-lang/rustup/actions/runs/3888788652/jobs/6636457603

It seems shellcheck can not use it as `env`.